packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -14,6 +14,10 @@ export class AmplifyProjectCreator {
   private readonly defaultDevPackages = [
     '@aws-amplify/backend',
     '@aws-amplify/backend-cli',
+    // TODO figure out how do we manage CDK version here as part of https://github.com/aws-amplify/amplify-backend/issues/392
+    'aws-cdk@2.110.1',
```

Comment:
Should we go ahead and bump to current latest?

Reply:
That's the plan after we implement testing.
For now I'd keep it at `2.110.1` , for the following reasons:
1. We didn't take dependency on any new API, so I think we shouldn't rush moving from `^2.110.1` in other packages. (narrowing range decreases flexibility for no good reason).
2. When move CDK version in create-amplify we may loose coverage of our "baseline supported cdk version", i.e. create amplify test would use "today's latest" and rest will use whatever package-lock provides. By keeping this as is we at least run some test with 2.110.1.

Ultimately we'd like to just install CDK with `^2` in create amplify flow, but before we do this we should implement the following guards:
1. Run tests with lowest CDK we support in `^` clause
1. Run tests with "latest cdk"
1. Perhaps add PR check to enforce that our packages in the repo declare consistent version of CDK (to simplify test matrix).